modin/engines/python/pandas_on_python/frame/partition.py

```diff
@@ -116,74 +179,119 @@ def to_pandas(self):
 
     def to_numpy(self, **kwargs):
         """
-        Convert the object stored in this partition to a NumPy array.
+        Return NumPy array representation of ``pandas.DataFrame`` stored in this partition.
+
+        Parameters
+        ----------
+        **kwargs : dict
+            Keyword arguments to pass into `pandas.DataFrame.to_numpy` function.
 
         Returns
         -------
-            A NumPy array.
+        np.ndarray
         """
         return self.apply(lambda df, **kwargs: df.to_numpy(**kwargs)).get()
 
     @classmethod
     def put(cls, obj):
-        """A factory classmethod to format a given object.
+        """
+        Return partition containing ``obj``.
```

Comment:
could you re-phrase that? the first impression of current doc is that something _searches_ for `obj` somewhere and returns a partition with that... maybe `create a partition` or smth